src/checkpoint/mod.rs

```diff
@@ -16,13 +16,16 @@ use tokio::select;
 use tokio::time::sleep;
 use tokio_graceful_shutdown::{IntoSubsystem, SubsystemHandle};
 
-pub use fvm::*;
+pub use bottomup::*;
 use ipc_identity::PersistentKeyStore;
+use ipc_sdk::subnet_id::SubnetID;
+pub use proof::create_proof;
 use std::fmt::Display;
 
+mod bottomup;
 mod fevm;
```

Comment:
Can't we remove all of these modules and maybe just have `evm` and `fvm` and choose the right checkpoint manager implementation according to the config? (I think this is where we are going once we refactor top-down, but just to double-check).

Reply:
yeah, that should be possible

Reply:
We should defer it for after the top-down checkpoint is in?